### ISSUE TITLE
recons = tf.reduce_sum(p_x.log_prob(x),1)

### ISSUE THREAD

**Issue by gitfourteen** [ericjang/gumbel-softmax#13]
Can you explain what `recons` stands for? Please not use acronyms or shorthand without annotation. Thank you!

**Comment by gitfourteen** [comment]
Should be **reconstruction loss** mentioned in [Auto-Encoding Variational Bayes](https://arxiv.org/abs/1312.6114); but found no reference to this jargon in neither paper on Gumbel Softmax
